docs/table/construct_table.rst

```diff
@@ -665,6 +679,43 @@ care because any modifications to the new Table data will also be seen in the
 original input data.  See the `Copy versus Reference`_ section for more
 information.
 
+rows
+----
+
+This argument allows providing data as a sequence of rows, in contrast
+to the ``data`` keyword which generally assumes data are a sequence of columns.
+The `Row data` section provides details.
+
+copy_indices
+------------
+
+If you are initializing a table from another table that has table
+indices defined, then this option allows copying that table *without* copying
+the indices by setting ``copy_indices=False``.  By default the indices are
+copied.
+
+units
+-----
+
+This allows setting the unit for one or more columns at the time of creating the
+table.  The input can be either a list of unit values corresponding to each of
+the columns in the table (using ``None`` for no unit), or a ``dict`` that
+provides the unit for specified column names.  For example::
+
+  >>> from astropy.table import QTable
+  >>> dat = [[1, 2] * u.m, ['hello', 'world']]
```

Comment:
In this example, probably best not to give the unit? At least, I would think that leads to a failure (cannot set a Quantity unit if it has one already).

Reply:
Done.